cypress/features/regression/button.feature

```diff
@@ -4,16 +4,6 @@ Feature: Button component
   Background: Open Button component default page
```

Comment:
Please, remove also those TC:
- Scenario: Set Button icon to arrow_left_small
- Scenario Outline: Change Button icon position to <iconPosition>

Reply:
removed